integration/fuse/src/main/java/alluxio/cli/metadatacache/DropAllCommand.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.cli.metadatacache;
+
+import alluxio.AlluxioURI;
+import alluxio.Constants;
+import alluxio.client.file.FileSystem;
+import alluxio.client.file.MetadataCachingBaseFileSystem;
+import alluxio.client.file.URIStatus;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.conf.PropertyKey;
+import alluxio.exception.status.InvalidArgumentException;
+import alluxio.cli.command.AbstractFuseShellCommand;
+import alluxio.wire.FileInfo;
+
+public final class DropAllCommand extends AbstractFuseShellCommand {
+  public DropAllCommand(FileSystem fs, AlluxioConfiguration conf, String parentCommandName) {
+    super(fs, conf, parentCommandName);
+  }
+
+  @Override
+  public String getCommandName() {
+    return "dropAll";
+  }
+
+  @Override
+  public String getUsage() {
+    return String.format("%s%s.%s.%s", Constants.DEAFULT_FUSE_MOUNT,
+        Constants.ALLUXIO_CLI_PATH, getParentCommandName(), getCommandName());
```

Comment:
We call getParentCommandName() which may return null multiple times, it's not good to use string.format with a null value. Can we reconsider how to deal with null cases?